src/main/java/org/codewars/utils/TaskRunner.java

```diff
@@ -3,17 +3,26 @@
 import java.math.BigInteger;
 
 public class TaskRunner {
+
+    private final Reader reader = new Reader();
+    private final Users user;
     public static void setUser(int id){
         // TODO: Implement setUser() method
     }
     public static long[] runGap() {
         return new long[0];
     }
 
-    public static BigInteger runPerimeter(){
-        return new BigInteger("0");
-    }
-
+//    public static BigInteger runPerimeter(){
+//        return new BigInteger("0");
+//    }
+public static void runPerimeter() {
+    System.out.println("Run task perimeter");
```

Comment:
mb use "Perimeter of squares in a rectangle" instead of method name?

Reply:
+